air/src/execution_step/lambda_applier/applier.rs

```diff
@@ -231,3 +303,22 @@ impl<'value> LambdaResult<'value> {
         }
     }
 }
+
+impl<'value> MapLensResult<'value> {
+    fn from_cow(result: Cow<'value, JValue>, tetraplet: RcSecurityTetraplet) -> Self {
```

Comment:
I'd call the method so simple just `new`.

Reply:
This breaks the naming schema used just above in impl of LambdaResult.